lib/Core/ErrorState.cpp

```diff
@@ -132,11 +132,9 @@ void ErrorState::outputErrorBound(llvm::Instruction *inst, double bound) {
     }
   }
 
-  stream << errorVar << " == (";
+  stream << "\nOutput Error: ";
   stream << PrettyExpressionBuilder::construct(e);
-  stream << ") && ";
-  stream << "(" << errorVar << " <= " << bound << ") && ";
-  stream << "(" << errorVar << " >= -" << bound << ")\n";
+  stream << "\nAbsolute bound: " << bound;
```

Comment:
@Himeshi Perhaps adding a newline (`<< "\n"`) here would make it easier to read?